### DISCUSSION TITLE
Displaying Album Artist Image

### DISCUSSION THREAD

**Discussion by shumancw** [jeffvli/feishin#196] [Q&A] [answered]
What is needed to fetch album artist? Does it come from an image file in an artist folder or does it come from Navidrome (in my case)? Mine are all showing as the default image.
![image](https://github.com/jeffvli/feishin/assets/7430301/c1c5773b-91fc-47fc-8568-8c677f402697)

**Comment by jeffvli** [comment, marked as answer]
You either need to [provide your own images](https://www.navidrome.org/docs/usage/artwork/) or use the [Spotify integration](https://www.navidrome.org/docs/usage/external-integrations/) on your Navidrome server.

**Reply by shumancw** [reply]
Thank you!
